### PR TITLE
Fix options is undefined

### PR DESCRIPTION
**Pull request by j1348**
it might missing some edge case, but it might fix the #62

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,9 @@ and in `<rootDir>/cypress/support/commands.js` add:
 ```js
 import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
 
-addMatchImageSnapshotCommand(options);
+Cypress.Commands.add('matchImageSnapshot', options => {
+    addMatchImageSnapshotCommand(options);
+});
```
Comment (crosscompile) [inline README.md]: I think you can actually just update this line in the README to:
```js
addMatchImageSnapshotCommand();
```

And then we don't need to change any code.

Comment (j1348) [reply README.md]: I already try it.

I think this is probably due to an Cypress API update regarding how commands are handle now https://docs.cypress.io/api/cypress-api/custom-commands.html

Comment (crosscompile) [reply README.md]: Hmm, I just ran [the example project](https://github.com/palmerhq/cypress-image-snapshot/tree/master/examples) which [calls `addMatchImageSnapshot` without `options`](https://github.com/palmerhq/cypress-image-snapshot/blob/master/examples/cypress/support/commands.js) and didn't run into any issues.

Are you able to run the example project locally?

Comment (j1348) [reply README.md]: I am going to try.

What is the Cypress version you had used ?

Comment (crosscompile) [reply README.md]: Okay sounds good, I used `cypress@3.1.4` and `cypress-image-snapshot@3.0.0`.

Comment (j1348) [reply README.md]: On my side, It seems that the examples test was failing with the `cypress@3.1.1` and now the examples test is succeeded with the `cypress@3.1.4`.

this sounds like a regression in between.

**Pull request closed without merging**